Makefile

```diff
@@ -168,12 +179,14 @@ KUSTOMIZE ?= $(LOCALBIN)/kustomize
 CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 ENVTEST ?= $(LOCALBIN)/setup-envtest
 GOLANGCI_LINT = $(LOCALBIN)/golangci-lint
+CRD_REF_DOCS ?= $(LOCALBIN)/crd-ref-docs
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v5.5.0
 CONTROLLER_TOOLS_VERSION ?= v0.16.4
 ENVTEST_VERSION ?= release-0.19
 GOLANGCI_LINT_VERSION ?= v1.61.0
+CRD_REF_DOCS_VERSION ?= v0.1.0
```

Comment:
It means that we need to manually maintain the tool's version. A better way is to leverage dependabot to automatically bump it. Refer to https://github.com/etcd-io/etcd/tree/main/tools/mod

But it's good enough in the very first step. We can continue to enhance it in next step.

Reply:
Now need to set the tool version manually now, can use dependabot to automatically bump it later.